### DISCUSSION TITLE
Is "bert_base_chinese" embedding model closed source?

### DISCUSSION THREAD

**Discussion by sunbuhui** [JohnSnowLabs/spark-nlp#2318] [Q&A]
```
...
val word_segmenter = WordSegmenterModel.pretrained("wordseg_large", "zh")
        .setInputCols(Array("sentence"))
        .setOutputCol("token")
val embeddings = BertEmbeddings.pretrained(name='bert_base_chinese', lang='zh')
        .setInputCols(Array("document", "token"))
        .setOutputCol("embeddings")
val ner = NerDLModel.pretrained("ner_msra_bert_768d", "zh")
        .setInputCols(["document", "token", "embeddings"])
        .setOutputCol("ner")
...
val pipeline = new Pipeline().setStages(Array(document_assembler, sentence_detector, word_segmenter, embeddings, ner, ner_converter))
val result = pipeline.fit(Seq.empty["马云在浙江省杭州市出生，是阿里巴巴集团的主要创始人。"].toDS.toDF("text")).transform(data)
```
I am using this code, but I can't download ``bert_base_chinese`` embedding model.
How can I get this model? 
Also, if ``bert_base_chinese`` model is not open source, models like ``ner_msra_bert_768d`` can't work, cause they use ``bert_base_chinese`` for embeddings. Put these models on the website may make people feel confused about why they can't work. If my guess is true, maybe they need docs to supplement.

**Comment by maziyarpanahi** [comment]
Hi,

This seems to be a bug. Could you please create an issue with the code and the full error so we can reproduce it?
